channel-layout.md

```diff
@@ -0,0 +1,131 @@
+---
+layout: default
+title: The Rust Release Channel Layout &middot; The Rust Programming Language
```

Comment:
```suggestion
title: The Rust Release Channel Layout
```

This way it's consistent with the other pages.

Reply:
I think I copied that from `release-process.md` but I'm happy to remove it.